src/frontend/src/assets/DormPriceCalculator.js

```diff
@@ -0,0 +1,116 @@
+/* All for request-admission.component.html */
```

Comment:
This logic would've been much simpler to do with Angular instead of DOM/JS, but I guess it's too late to change now (as long as it works :) )